Mod2/Lessons/Week5/CFUReview.md

```diff
@@ -0,0 +1,50 @@
+# Mod 2 Week 5 Checks for Understanding
+
+1. What are some reasons for using software versioning?
```

Comment:
I'm not sure I understand what this question is trying to get at - do we want students to discuss why different versions exist?  Or, why a development team would choose a specific version?